subsys/bluetooth/host/hci_core.c

```diff
@@ -4913,6 +4971,11 @@ static const struct event_handler prio_events[] = {
 	EVENT_HANDLER(BT_HCI_EVT_NUM_COMPLETED_PACKETS,
 		      hci_num_completed_packets,
 		      sizeof(struct bt_hci_evt_num_completed_packets)),
+#if defined(CONFIG_BT_REMOTE_VERSION)
+	EVENT_HANDLER(BT_HCI_EVT_REMOTE_VERSION_INFO,
+		      bt_hci_evt_read_remote_version_complete,
+		      sizeof(struct bt_hci_evt_remote_version_info)),
+#endif /* CONFIG_BT_REMOTE_VERSION */
```

Comment:
The handler should either be in `normal_events[]` or in `prio_events[]` but not both. I guess you're "deprioritizing" it along with this PR, so the above chunk should be removed.

Reply:
Thanks. That is an oversight. Will remove.